### PR TITLE
Add info about grafana, service file and build info

### PR DESCRIPTION
**Pull request by ochorocho**
See #5

Added information about Grafana and other stuff i needed to get it working.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -38,3 +38,50 @@ irate(wireguard_peer_receive_bytes_total[1m]) * on (public_key) group_left(name)
 ```
 irate(wireguard_peer_transmit_bytes_total[1m]) * on (public_key) group_left(name) wireguard_peer_info * on (instance) group_left(device) wireguard_device_info
 ```
+
+## Grafana Dashboard
+
+You can view your data using this [Grafana Dashbaord](https://grafana.com/grafana/dashboards/12177) using Prometheus as source.
```
Comment (mdlayher) [inline README.md:44]: *Dashboard

File: README.md [hunk 2]
```diff
@@ -38,3 +38,50 @@ irate(wireguard_peer_receive_bytes_total[1m]) * on (public_key) group_left(name)
...
+cd cmd/wireguard_exporter/
+go build .
+mv wireguard_exporter /usr/local/bin/
+```
+
+## Add service file for systemd
+
+```
+[Unit]
+Description=Prometheus Wireguard Exporter
```
Comment (mdlayher) [inline README.md:60]: Nit: "WireGuard" is proper capitalization

File: README.md [hunk 3]
```diff
@@ -38,3 +38,50 @@ irate(wireguard_peer_receive_bytes_total[1m]) * on (public_key) group_left(name)
...
+Load new service and enable autostart
+
+```
+systemctl daemon-reload
+systemctl enable wireguard-exporter.service
+```
+
+## Add scraping config to prometheus
+
+In `/etc/prometheus/prometheus.yml` add following config to the secion `scrape_configs:` :
```
Comment (mdlayher) [inline README.md:81]: *section

File: README.md [hunk 4]
```diff
@@ -38,3 +38,50 @@ irate(wireguard_peer_receive_bytes_total[1m]) * on (public_key) group_left(name)
...
+[Service]
+Type=simple
+Restart=always
+ExecStart=/usr/local/bin/wireguard_exporter
+
+[Install]
+WantedBy=multi-user.target
+```
+
+Load new service and enable autostart
```
Comment (mdlayher) [inline README.md:72]: Add a colon here too.

**Comment by ochorocho** [comment]
@mdlayher thanks for the review. :-) 

Changed/fixed stuff you suggested.

**Pull request merged**